### PR TITLE
Bump jQuery and jQuery-ui versions

### PR DESCRIPTION
**Pull request by jvoisin**
I bumped it first to 2.X with jQuery migrate, played around but didn't manage
to trigger any warning.  So I bumped it again to the latest available version,
jQuery 3.4.0, which isn't triggering useful warnings either.

### CODE REVIEW

**Comment by jvoisin** [comment]
The build failure is due to some CVE in Spring :/

**Comment by jvoisin** [comment]
I played a bit around but didn't manage to find anything broken, UI-wise, but it would be nice if someone else could test this PR too.

**Comment by jooola** [comment]
Could you share me the war file of this PR ?

**Comment by jvoisin** [comment]
[here](https://send.firefox.com/download/ec8755cef3120f03/#JlECMU8ou485P_uG04XJ_g) you go.

**Comment by jvoisin** [comment]
Don't forget to squash this PR before merging it, so that we don't have useless files in the history.

**Comment by muff1nman** [comment]
@jvoisin @jooola how extensively have you tested this?

**Comment by jooola** [comment]
Havn't had time for this...

**Comment by jvoisin** [comment]
I've played with it on Firefox and Chrome, and haven't noticed anything weird.

File: airsonic-main/src/main/webapp/WEB-INF/jsp/jquery.jsp [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<link rel="stylesheet" href="<c:url value="/style/smoothness/jquery-ui-1.8.18.custom.css"/>" type="text/css">
+<link rel="stylesheet" href="<c:url value="/style/smoothness/jquery-ui.min.css"/>" type="text/css">
```
Comment (muff1nman) [inline airsonic-main/src/main/webapp/WEB-INF/jsp/jquery.jsp]: Can we keep the version number here?

**Pull request merged**